README.md

```diff
@@ -1,157 +1,101 @@
+5. Fix linter errors.
+6. **IMPORTANT NOTE**: feel free to research [auto-correct options for Stylelint](https://stylelint.io/user-guide/usage/options) if you get a flood of errors but keep in mind that correcting style errors manually will help you to make a habit of writing a clean code!
 
-<p align="right">(<a href="#readme-top">back to top</a>)</>
+### [ESLint](https://eslint.org/)
 
-## 🚀 Live Demo <a name="live-demo"></a>
+1. Run 
+   ```
+   npm install --save-dev eslint@7.x eslint-config-airbnb-base@14.x eslint-plugin-import@2.x babel-eslint@10.x
+   ``` 
+   *not sure how to use npm? Read [this](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm).*
 
-
-
-- [Live Demo Link]( https://danielochuba.github.io/Portfolio-setup-and-mobile-first/)
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-
-
-## 💻 Getting Started <a name="getting-started"></a>
-
-
-To get a local copy up and running, follow these steps.
-### Prerequisites
-
-In order to run this project you need:
-
-<ul>
-  <li>A code editor like Visual Studio Code</li>
-  <li>Internet connection</li>
-  <li>A working browser</li>
-</ul>
-
-### Setup
-  <ol>
-    <li>Copy the URL for the repository.
-      <li>To clone the repository using HTTPS, under "HTTPS"</li>
-      <li>To clone the repository using an SSH key, including a certificate issued by your organization's SSH certificate authority, click SSH    </li>
-      <li>To clone a repository using GitHub CLI, click GitHub CLI</li>
-    </li>
-    <li>Open Git Bash.</li> 
-    <li>Change the current working directory to the location where you want the cloned directory.</li>
-    <li>Type git clone, and then paste the URL you copied earlier.<br><code>$ git clone https://github.com/YOUR-USERNAME/YOUR-REPOSITORY</code></li>
-    <li>Press Enter to create your local clone</li>
-  </ol>
-
-### Install
-
-Install this project with:
-
-  sh Portfolio: set up mobile first install
-
-### Usage
-
-To run the project, execute the following command:
-.code index.html
-
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-
-
-## 👥 Authors <a name="authors"></a>
-
-
-
-👤 **Author1**
-
-- GitHub: [@githubhandle](https://github.com/danielochuba)
-- Twitter: [@twitterhandle](https://twitter.com/ochuba_daniel)
-- LinkedIn: [LinkedIn](www.linkedin.com/in/daniel-ochuba-614572238)
-
-
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-
-
-## 🤝 Contributing <a name="contributing"></a>
-
-Contributions, issues, and feature requests are welcome!
-
-Feel free to check the [issues page](../../issues/).
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-
-## ⭐️ Show your support <a name="support"></a>
-
-
-
-If you like this project you can give me a star
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-
-
-## 📝 License <a name="license"></a>
-
-This project is [MIT](./MIT.md) licensed.
-
```

Comment:
- [ ] When I click the link to the `MIT.md` file that you added, I get this 404 `error`:

![00Mit](https://user-images.githubusercontent.com/79058364/182822618-fdfb2523-cb23-4060-9f72-75ec8c29e4cd.png)


#### The reason for this is that there is no `MIT.md` file in the root directory of this project. Kindly attach the file. 👍

- [ ] Please update your Readme, it will be nice to add and update this section(`License`) with the type of License and a link to the license file, as this will make your readme more informative and professional. Check out this [template](https://github.com/microverseinc/readme-template) for more clarifications
-  Please, consider fixing the broken link to the MIT.md file 🥇